### PR TITLE
Add FAQ about Testnet

### PR DESCRIPTION
**Pull request by RiccardoMasutti**

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -1034,6 +1034,20 @@ At least one round to re-mix your coins can increase your privacy drastically.
...
+The testnet is an alternative Bitcoin blockchain, to be used for testing.
+Testnet coins (tBTC) are separate and distinct from actual bitcoins, and are never supposed to have any value.
+This allows application developers or bitcoin testers to experiment, without having to use real bitcoins or worrying about breaking the main bitcoin chain.
+You can try to make a [CoinJoin with Wasabi on the Bitcoin TestNet](/FAQ/FAQ-UseWasabi.html#i-d-like-to-experience-coinjoin-but-i-m-not-comfortable-using-real-bitcoin-what-can-i-do) without being afraid of losing 'real' bitcoins.
+
+There have been three generations of testnet.
+Testnet2 was just the first testnet reset with a different genesis block, because people were starting to trade testnet coins for real money.
+Testnet3 is the current test network.
+It was introduced with the 0.7 release, introduced a third genesis block, a new rule to avoid the "difficulty was too high, is now too low, and transactions take too long to verify" problem, and contains blocks with edge-case transactions designed to test implementation compatibility.
+:::
```
Comment (molnard) [inline docs/FAQ/FAQ-UseWasabi.md]: Idea: add some links where a user can get TestNet coins?

Comment (RiccardoMasutti) [reply docs/FAQ/FAQ-UseWasabi.md]: Done

**Comment by yahiheb** [comment]
@MaxHillebrand Please be careful when resolving conflicts, the merges are making a mess.

I don't think we should add questions about what is Testnet, this is a Bitcoin related question and the Wasabi Docs should contain Wasabi related questions (maybe some bitcoin questions but not everything). Should we then add a question about what is Regtest? or what is lightning?
